trl/trainer/cpo_config.py

```diff
@@ -26,7 +26,7 @@ class CPOConfig(TrainingArguments):
     [argparse](https://docs.python.org/3/library/argparse#module-argparse) arguments that can be specified on the
     command line.
 
-    Parameters:
+    Args:
```

Comment:
```suggestion
    Parameters:
```

For configs (only) I'd keep `Parameters` to be consistent with `TrainingArguments`. Even though I agree that `Args` is more aligned with the convention.

Reply:
Sounds good, I'll change it then.

Reply:
Done :)